056-CosmicTroubleshooting/Student/Challenge-00.md

```diff
@@ -0,0 +1,88 @@
+  - [Azure Portal](../../000-HowToHack/WTH-Common-Prerequisites.md#azure-portal)
+  - [Azure Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell) - v7.2 or newer required
+    - [Azure PowerShell CmdLets](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-8.2.0)
+  - [Azure CLI](../../000-HowToHack/WTH-Common-Prerequisites.md#azure-cli)
+    - [Note for Windows Users](../../000-HowToHack/WTH-Common-Prerequisites.md#note-for-windows-users)
+    - [Azure PowerShell CmdLets](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-8.2.0)
+  - [Azure Cloud Shell](../../000-HowToHack/WTH-Common-Prerequisites.md#azure-cloud-shell)
+- [Visual Studio Code](../../000-HowToHack/WTH-Common-Prerequisites.md#visual-studio-code)
+  - [VS Code plugin for ARM Templates](../../000-HowToHack/WTH-Common-Prerequisites.md#visual-studio-code-plugins-for-arm-templates)
+  - [VS Code plugin for Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep)
+- [Azure Storage Explorer](../../000-HowToHack/WTH-Common-Prerequisites.md#azure-storage-explorer)
+
+## Description
+
+Now that you have the common pre-requisites installed on your workstation, there are prerequisites specific to this hack.
+
+Your coach will provide you with a Resources.zip file that contains resources you will need to complete the hack. If you plan to work locally, you should unpack it on your workstation. If you plan to use the Azure Cloud Shell, you should upload it to the Cloud Shell and unpack it there.
+
+Please install these additional tools:
+
+- .NET SDK 6.0 or later installed on your development machine. This can be downloaded from [here](https://www.microsoft.com/net/download/all) for multiple platforms.
+
+In the `/Challenge00/` folder of the Resources.zip file given by your coach, two deployment scripts that deploy a bicep template with the required services for the challenge, as well as building and deploying our sample web application.
+
+Please deploy the infrastructure by running the following scripts in the `/Challenge00/` folder:
+
+- Powershell: 
+  ``` 
+  # Update your Az Powershell commandlets
+  Update-Module Az
+
+  # Connect to your Azure Account
+  Connect-AzAccount
+
+  # Deploy the infrastructure
+  # Please make sure you are using Powershell v7.2 or newer
+  # You might need to unblock the file
+  .\deploy.ps1 
+  ```
+- Bash:
+  ```
+  # Make sure your machine has the prerequisites
+  # Also does an "az login"
+  ./setup-machine.sh
+
+  # Deploy the infrastructure
+  ./deploy.sh
+  ```
+
+## Success Criteria
+
+
+To complete this challenge successfully, you should be able to:
+
+- The deployment script (either Powershell or Bash) has completed successfully
+- Verify that the deployment script has deployed in your subscription under a Resource Group call `rg-wth-azurecosmosdb` (if left at the default value) the following resources:
```

Comment:
Change to "The deployment script has deployed in your subscription under a Resource Group called `rg-wth-azurecosmosdb` (if left at the default value) and it contains the following resources:"

Reply:
Thank you Pete! Changed as per above.